completion.go

```diff
@@ -77,10 +78,21 @@ func checkEndpointSupportsModel(endpoint, model string) bool {
 	return !disabledModelsForEndpoints[endpoint][model]
 }
 
+func checkPromptType(prompt any) (ok bool) {
```

Comment:
Sorry, I couldn't help myself not to do some code golfing here :D  https://go.dev/play/p/-2aR0tII8xG

```go
func checkPromptType(prompt any) bool {
	_, isString := prompt.(string)
	_, isStringSlice := prompt.([]string)
	return isString || isStringSlice
}
```

Reply:
@NullpointerW I've approved the PR, you can either leave your implementation or add this one — up to you!